packages/@glimmer/compiler/test/compiler-test.ts

```diff
@@ -55,6 +55,19 @@ function test(desc: string, template: string, ...expectedStatements: BuilderStat
 const Append = Builder.Append;
 const Concat = Builder.Concat;
 
+QUnit.test(
```

Comment:
this doesn't match the `test` style in the rest of the file, but i thought it might be more clear to just use `QUnit.test` directly rather than extend the `test` function in this file to allow a callback/expect errors. Totally open to change this to whatever the maintainers prefer, let me know!